### PR TITLE
Create fedramp-faq.md

### PR DESCRIPTION
**Pull request by elstudio**
Create a hidden page from our FedRAMP FAQ content as described here: https://github.com/github/government/pull/448

### CODE REVIEW

**Comment by elstudio** [comment]
@jbjonesjr I think this looks ready to roll.

Travis thinks that this page is a broken link (since it exists only on the PR branch, I guess). One day I will look into that.

However today, we have an actual page with FedRAMP copy: https://github-5a58e98664.drafts.github.io/government.github.com/fedramp-faq

/cc @sophshep This is a hidden page, so hopefully I haven't done something too ugly here. :)

/cc @anweiss FYI

File: fedramp-faq.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: support-page
+title: FedRAMP and GitHub
+description: GitHub Enterprise Cloud is now FedRAMP Authorized, which brings software collaboration to governments everywhere
+permalink: /fedramp-faq
+
+---
+
+
+# 🎉GitHub Enterprise Cloud + FedRAMP Tailored Authorized 🎉
```
Comment (jbjonesjr) [inline fedramp-faq.md]:
```suggestion
# 🎉&nbsp;GitHub Enterprise Cloud + FedRAMP Tailored Authorized 🎉
```

Comment (jbjonesjr) [reply fedramp-faq.md]: seems scrunched

File: fedramp-faq.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Governments around the world use GitHub to build software, shape policy, and share information with constituents. To ensure that governments can have access to best-in-class tools, we’ve worked with the US Federal Government to secure FedRAMP Tailored Authorization for GitHub Enterprise Cloud. 
+Government users can now rely on GitHub knowing that our platform meets the FedRAMP Tailored baseline of security standards set by our US federal government partners.
+
+With GitHub Enterprise Cloud’s FedRAMP Tailored ATO, agencies can more easily:
+
+- Securely collaborate on software development and management in the cloud
+- Modernize the way they build software
+- Take advantages of the latest features and capabilities that GitHub has to offer
+- Jump start agency authorization by reviewing our current FedRAMP Tailored ATO and authorization package
+
```
Comment (jbjonesjr) [inline fedramp-faq.md:21]:
```suggestion
<br/>
```
For some reason this butted up to the next header

**Review by jbjonesjr — CHANGES_REQUESTED** [review]
minor nits

**Comment by sophshep** [comment]
> /cc @sophshep This is a hidden page, so hopefully I haven't done something too ugly here. :)

I've moved over to product design, so I no longer have this site under my AoR. cc/ @github/site-design who can help review!

File: fedramp-faq.md (hunk 1 above)
Comment (elstudio) [reply fedramp-faq.md]: 👍

File: fedramp-faq.md (hunk 2 above)
Comment (elstudio) [reply fedramp-faq.md:21]: Let's do this thing.

**Review by elstudio — COMMENTED** [review]
A little white space never hurt anybody. Thanks for these, @jbjonesjr!

**Review by jbjonesjr — APPROVED** [review]

**Pull request merged**